src/core/ext/filters/client_channel/config_selector.h

```diff
@@ -80,10 +80,8 @@ class DefaultConfigSelector : public ConfigSelector {
 
   CallConfig GetCallConfig(GetCallConfigArgs args) override {
     CallConfig call_config;
-    if (service_config_ != nullptr) {
-      call_config.method_configs =
-          service_config_->GetMethodParsedConfigVector(*args.path);
-    }
+    call_config.method_configs =
```

Comment:
Can we add 
`GPR_DEBUG_ASSERT(service_config_ != nullptr)` here with a comment as to why it will never happen?

Reply:
Done.